src/components/Offcanvas/docs/Offcanvas.mdx

```diff
@@ -0,0 +1,79 @@
+---
+name: Offcanvas
+menu: Components
+route: /components/Offcanvas
+---
+
+import { Offcanvas } from '../Offcanvas.tsx';
+import { OffcanvasCreator } from './OffcanvasCreator.tsx';
+import { OffcanvasPropsTable } from './OffcanvasPropsTable.tsx';
+import { StyledSystemLinks } from '../../../docs/StyledSystemLinks'
+
+# Offcanvas
+
+The Offcanvas should only contain content or flows which are related to the current page and not replace pages, therefore
+its complexity level has to be simple. Consider the fullscreen state if you want to focus the users attention to the Offcanvas
+content.
+
+Offcanvas is responsive per default but the width can be adjusted (the default is 600px or 37.5rem). Content on the Offcanvas
+is scrollable and has a space5 bounding box.
```

Comment:
Maybe we need a different text here, do we have some in charge for this particular task?

Reply:
We need to review this. @agonzalezUX @rafael-sepeda can you read this documentation?
Sadly we yet don't have preview links so can you read this file?